airsonic-main/src/main/java/org/airsonic/player/service/VersionService.java

```diff
@@ -193,16 +193,11 @@ private String readLineFromResource(String resourceName) {
         if (in == null) {
```

Comment:
Maybe it would be worth trying to move this function and combine it in a more generic form with the very similar `readLines()` in StringUtil.java?